account_move_base_import/models/account_journal.py

```diff
@@ -89,6 +105,64 @@ def _find_values_from_rules(self, calls, line):
         return None
 
     @api.multi
+    def _prepare_counterpart_line(self, move, amount, date):
+        if amount > 0.0:
+            account_id = self.default_debit_account_id.id
+            credit = 0.0
+            debit = amount
+        else:
+            account_id = self.default_credit_account_id.id
+            credit = -amount
+            debit = 0.0
+        counterpart_values = {
+            'name': _('/'),
+            'date_maturity': date,
+            'credit': credit,
+            'debit': debit,
+            'partner_id': self.partner_id.id,
+            'move_id': move.id,
+            'account_id': account_id,
+            'already_completed': True,
+            'journal_id': self.id,
+            'company_id': self.company_id.id,
+            'currency_id': self.currency_id.id,
```

Comment:
Yesterday, there was a small change in the counterpart : https://github.com/OCA/bank-statement-reconcile/pull/137/commits/62f2cfe6ee4bce04042d9cfbd807b7e053051c2b ; you need the field "amount_residual" in the counterpart.